src/ManageCourses.Api/appsettings.Development.json

```diff
@@ -8,10 +8,11 @@
       }
     }
   },
-  "PG_USERNAME":"manage_courses",
-  "PG_PASSWORD":"manage_courses",
-  "auth:oidc:userinfo_endpoint":"https://signin-test-oidc-as.azurewebsites.net/me",
-  "api:key":"chicken",
+  "PG_USERNAME": "manage_courses",
+  "PG_PASSWORD": "manage_courses",
+  "auth:oidc:userinfo_endpoint": "https://signin-test-oidc-as.azurewebsites.net/me",
+  "api:key": "chicken",
+  "manage_courses_backend:key": "rubbish-ruby",
```

Comment:
Can you change this to `bananas` please?

Reply:
Sure fat `Banana` it be than.